index.html

```diff
@@ -63,5 +63,43 @@ <h1>
     
         </div>
     </section>
+
+   <!-- ------------------------------------mobile section-------------------- -->
+<section class="section-nav">
+
+   <header class="header-mobile">
+    <nav>
+        <img src="images/section_header.png" alt="logo">
+    </nav>
+   </header>
+
+<section class="section-mobile">
+   <div class="project-title">
+    <h1>Tonic</h1>
+    <ul class="top-list">
+        <li>CANOPY</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+        <li>Back End Dev</li>
+        <li><img src="images/Counter.png" alt="dot-counter"></li>
+    </ul>
+   </div>
+    <p>
+        A daily selection of privately personalized 
+        reads; no accounts or 
+        sign-ups required.
+    </p>
+
+    <ul class="list-bottom">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <div class="action">
+      <button>See Project</button>
+    </div>
+    
+
+</section>
+</section>
 </body>
```

Comment:
- [ ] Please make sure to add the missing section of your portfolio, according to your requirements you need to add the "About me" section matching the styles of your Figma template:
![image](https://user-images.githubusercontent.com/52635378/170833718-5f3de090-d279-49de-9698-2013ff8a9dee.png)